### ISSUE TITLE
Retina Images in Cache

### ISSUE THREAD

**Issue by mheicke** [onevcat/Kingfisher#70]
Hi,
I think there's a problem with retrieving cached images on retina displays. In the diskImageForKey function in the ImageCache extension, the image is retrieved by using
```swift
    let image = UIImage(data: data)
```
So the scale of the device is not considered as it is in the URLSession(session: NSURLSession, task: NSURLSessionTask, didCompleteWithError error: NSError?) function in ImageDownloader.swift.

I think it should be 
```swift
    let image = UIImage(data: data, scale: UIScreen.mainScreen().scale)
```

Maybe it would be nice, if the scaling would be optional, nevertheless I think it should be the same in ImageCache and ImageDownloader.

Keep up with the good work,
matt

**Comment by onevcat** [comment]
Sure, you are right. I will fix it soon.

**Comment by onevcat** [comment]
Resolved in #77
